apps/server/src/modules/h5p-editor/controller/dto/h5p-editor.params.ts

```diff
@@ -16,6 +16,20 @@ export class GetH5PContentParams {
 	contentId!: string;
 }
 
+export class GetH5PEditorParams {
+	@ApiProperty()
+	@Matches('([a-z]+-[a-z]+)')
+	@IsString()
+	@IsOptional()
```

Comment:
can this be an enum instead of any string?

Reply:
Might be. Currently language is not used at all, might be removed from the query later. Will be part of a later ticket.